### ISSUE TITLE
Add JSI Tests in repo

### ISSUE THREAD

**Issue by dannyvv** [microsoft/hermes-windows#77]
## Problem

We need to add tests to this repo

## Solution

Update `.ado/jobs.yml` to have tests run and remove the step with `[Test] TODO`

**Comment by mganandraj** [comment]
@vmoroz I believe you have enabled a few tests. Should we close this bug ?

**Comment by vmoroz** [comment]
@mganandraj , no, I am able to run the tests locally on my machine, but I did not make them part of the CI yet. Let's keep this issue opened.

**Comment by vmoroz** [comment]
Closing this issue as the unit tests were enabled in this repo as a part of the build pipeline redesign.
